_sass/messages.scss

```diff
@@ -12,11 +12,19 @@ $error: #BF433A;
   background: transparentize($info, 0.8);
   border-color: $info;
   color: $info;
+
+  p {
```

Comment:
Might be worth a comment here (and in `.warning`) about why this is needed?

Reply:
👍 Sounds sensible, will add a comment

Reply:
a25d6f4